spec/fixtures/mocks/submission_completed.json

```diff
@@ -10,53 +10,6 @@
       "purchase_order_number": null,
       "status": "completed",
       "levy": 4500
-    },
-    "relationships": {
-      "entries": {
-        "data": [
-          {
-            "type": "submission_entries",
-            "id": "f87717d4-874a-43d9-b99f-c8cf2897b526"
-          },
-          {
-            "type": "submission_entries",
-            "id": "1eb3b8ee-0ac6-4b8f-86a5-6886b33c63ff"
-          },
-          {
-            "type": "submission_entries",
-            "id": "32423310-e3b6-4e2f-b022-a4854d8085ab"
-          }
-        ]
-      }
     }
-  },
-  "included": [
-    {
-      "id": "f87717d4-874a-43d9-b99f-c8cf2897b526",
-      "type": "submission_entries",
-      "attributes": {
-        "source": { "row": 42, "sheet": "InvoicesRaised" },
-        "data": { "test": "test" },
-        "status": "validated"
-      }
-    },
-    {
-      "id": "32423310-e3b6-4e2f-b022-a4854d8085ab",
-      "type": "submission_entries",
-      "attributes": {
-        "source": { "row": 40, "sheet": "Invoices Raised" },
-        "data": { "test": "test" },
-        "status": "validated"
-      }
-    },
-    {
-      "id": "1eb3b8ee-0ac6-4b8f-86a5-6886b33c63ff",
-      "type": "submission_entries",
-      "attributes": {
-        "source": { "row": 1, "sheet": " Orders  Received " },
-        "data": { "test": "test" },
-        "status": "validated"
-      }
-    }
-  ]
```

Comment:
Mocking question, mostly for my understanding – we assert that this isn't coming back from the API by deleting these lines, and I don't know enough about the JSON API consumer to know – I assume that removing `has_many :entries` and `errored_entries` above effects a result that resembles this deletion. Would that be right?

Reply:
I'm not sure I fully understand the question, but removing these elements from the JSON is to make the fixtures JSON resemble what the API will actually be supplying.

Reply:
What I meant was: I don't see anything in the API PR that effects the removal of the entries, so have we simply stopped asking for them from the client?

Reply:
ah, I understand. This is a good question, technically, because the the `SerializableSubmission` class still `has_many :entries` defined, then a request could be made to retrieve a submission and all its entries. we should probably remove that relationship from the serialisable model to make this explicitly not possible.

Reply:
It was terribly phrased though. Sorry about that.